setup.py

```diff
@@ -55,7 +57,7 @@ def parse_requirements(path):
     function properly.
     """
     try:
-        requirements = map(str.strip, local_file(path).splitlines())
+        requirements = [req.strip() for req in local_file(path).splitlines()]
```

Comment:
Now that `local_file` decodes the file content, it will return a `unicode` in Python 2 and a `str` in Python 3; mapping `str.strip` causes problems:

    TypeError: descriptor 'strip' requires a 'str' object but received a 'unicode'